sdk/hybridnetwork/Azure.ResourceManager.HybridNetwork/src/Generated/Models/ArtifactStorePropertiesFormat.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.HybridNetwork.Models
+{
+    /// <summary> Artifact store properties. </summary>
+    public partial class ArtifactStorePropertiesFormat
+    {
+        /// <summary> Initializes a new instance of ArtifactStorePropertiesFormat. </summary>
+        public ArtifactStorePropertiesFormat()
+        {
+        }
+
+        /// <summary> Initializes a new instance of ArtifactStorePropertiesFormat. </summary>
+        /// <param name="provisioningState"> The provisioning state of the application groups resource. </param>
+        /// <param name="storeType"> The artifact store type. </param>
+        /// <param name="replicationStrategy"> The replication strategy. </param>
+        /// <param name="managedResourceGroupConfiguration"></param>
+        /// <param name="storageResourceId"> The created storage resource id. </param>
+        internal ArtifactStorePropertiesFormat(ProvisioningState? provisioningState, ArtifactStoreType? storeType, ArtifactReplicationStrategy? replicationStrategy, ArtifactStorePropertiesFormatManagedResourceGroupConfiguration managedResourceGroupConfiguration, string storageResourceId)
+        {
+            ProvisioningState = provisioningState;
+            StoreType = storeType;
+            ReplicationStrategy = replicationStrategy;
+            ManagedResourceGroupConfiguration = managedResourceGroupConfiguration;
+            StorageResourceId = storageResourceId;
+        }
+
+        /// <summary> The provisioning state of the application groups resource. </summary>
+        public ProvisioningState? ProvisioningState { get; }
+        /// <summary> The artifact store type. </summary>
+        public ArtifactStoreType? StoreType { get; set; }
+        /// <summary> The replication strategy. </summary>
+        public ArtifactReplicationStrategy? ReplicationStrategy { get; set; }
+        /// <summary> Gets or sets the managed resource group configuration. </summary>
+        public ArtifactStorePropertiesFormatManagedResourceGroupConfiguration ManagedResourceGroupConfiguration { get; set; }
+        /// <summary> The created storage resource id. </summary>
+        public string StorageResourceId { get; }
```

Comment:
What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://github.com/Azure/azure-sdk-for-net/blob/bd953fc7a97bf617c892ce6314b50126b4021a4f/sdk/maps/Azure.ResourceManager.Maps/src/autorest.md?plain=1#L25).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://github.com/Azure/azure-sdk-for-net/blob/bd953fc7a97bf617c892ce6314b50126b4021a4f/sdk/search/Azure.Search.Documents/src/autorest.md?plain=1#L106-L115).